cmdline-pull/pubsub_sample.py

```diff
@@ -105,6 +108,30 @@ def create_topic(client, args):
     print 'Topic {} was created.'.format(topic['name'])
 
 
+def grant_gmail_push(client, args):
+    """Grants gmail push notification on this topic."""
```

Comment:
What does this grant? Would a better docstring be """Grants the gmail push notification service account publish access to this topic."""